src/modules/units/components/UnitPage.tsx

```diff
@@ -0,0 +1,106 @@
+import { Box, Paper } from '@mui/material';
+import React, { useEffect, useMemo } from 'react';
+import CommonTabs from '@/components/elements/CommonTabs';
+import Loading from '@/components/elements/Loading';
+import PageTitle from '@/components/layout/PageTitle';
+import NotFound from '@/components/pages/NotFound';
+import useSafeParams from '@/hooks/useSafeParams';
+
+import PrioritizedClientsTable from '@/modules/ce/components/unit/PrioritizedClientsTable';
+import UnitReferralStatus from '@/modules/ce/components/UnitReferralStatus';
+import { useProjectDashboardContext } from '@/modules/projects/components/ProjectDashboard';
+import UnitOverview from '@/modules/units/components/UnitOverview';
+import { ProjectDashboardRoutes } from '@/routes/routes';
+import { useGetUnitQuery } from '@/types/gqlTypes';
+
+interface Props {}
+const UnitPage: React.FC<Props> = ({}) => {
+  const { unitId } = useSafeParams() as {
+    unitId: string;
+    projectId: string;
+  };
+
+  const { project, overrideBreadcrumbTitles } = useProjectDashboardContext();
+
+  const {
+    data: { unit } = {},
+    loading,
+    error,
+  } = useGetUnitQuery({
+    variables: {
+      id: unitId,
+      includeCeFields: true, // we only render this page if CE is enabled
+    },
+  });
+
+  // Set the breadcrumb so it says the correct name of this unit and group
+  useEffect(() => {
+    if (!unit) return;
+
+    overrideBreadcrumbTitles({
+      [ProjectDashboardRoutes.UNIT]: unit.name,
+      // fixme unable to inject unit group name
```

Comment:
Fixme flag - do you want to solve this on this PR? I think it is ok to leave for later. Maybe we should start a catchall ticket for design/bug bash followups?

Reply:
I started collecting a few more of these, so I created https://github.com/open-path/Green-River/issues/7799 for a place to put my notes somewhere shareable.

Feel free to remove the note about this one on that ticket, if you decide to address it in this PR!